src/Interface.jl

```diff
@@ -42,15 +40,15 @@ function cooccurring(com::AbstractAssemblage, inds::AbstractVector)
     richness(sub) .== nthings(sub)
 end
 
-function createsummaryline(vec::AbstractVector{T}) where T<:AbstractString
-    linefunc(vec) = mapreduce(x->x*", ", *, vec[1:(end-1)])*vec[end]
+function createsummaryline(vec::AbstractVector{T}) where T <: AbstractString
+    linefunc(vec) = mapreduce(x->x * ", ", *, vec[1:(end-1)]) * vec[end]
```

Comment:
Recommended style is to have spaces around `+`, `-` but none around `*`, `/`, `:` etc. I don't care much.

Reply:
Okay, I didn't know that, thanks.